### PR TITLE
feat: from product trial site list page redirect user to product page as per config

### PR DESCRIPTION
**Pull request by tanmoysrt**

### CODE REVIEW

File: dashboard/src2/pages/NewAppTrial.vue [hunk 1]
```diff
@@ -272,6 +272,18 @@ export default {
 				name: this.productId
 			};
 		},
+		site() {
+			return {
+				type: 'document',
+				method: 'get_product_redirect_url',
```
Comment (BreadGenie) [inline dashboard/src2/pages/NewAppTrial.vue:278]: don't need method for document resource 
https://frappeui.com/story/docs-resources-document-resource-story-js#options

File: dashboard/src2/pages/NewAppTrial.vue [hunk 2]
```diff
@@ -342,6 +354,11 @@ export default {
 			let cluster = await this.getClosestCluster();
 			return this.siteRequest.createSite.submit({ cluster });
 		},
+		async redirectToProductPage(site) {
+			window.location.href = await this.site.productRedirectUrl.submit({
+				site
+			});
```
Comment (BreadGenie) [inline dashboard/src2/pages/NewAppTrial.vue:360]: site argument isn't required as well

**Review by BreadGenie — CHANGES_REQUESTED** [review]

File: dashboard/src2/pages/NewAppTrial.vue (hunk 1 above)
Comment (tanmoysrt) [reply dashboard/src2/pages/NewAppTrial.vue:278]: okay will update it.

**Comment by tanmoysrt** [comment]
It's not required anymore.

**Pull request closed without merging**